packages/common/src/kms.ts

```diff
@@ -14,13 +14,13 @@ export class KMS {
     };
 
     const kms = new AWS.KMS();
-    const { CiphertextBlob } = await kms.encrypt(params).promise();
+    const CiphertextBlob = await kms.encrypt(params);
 
     if (!CiphertextBlob) {
       throw new Error('Encryption failed, undefined CiphertextBlob returned');
     }
 
-    return CiphertextBlob.toString('base64');
```

Comment:
Why was this removed, or paerhaps more to the point why was it there in the first place?    I'm concerned with the `any` type coming back from .encrypt and this change with respect to why it was needed.